Source/WTF/wtf/text/AtomString.h

```diff
@@ -97,12 +97,12 @@ class AtomString final {
     // If we need more overloads of the number function, we can add all the others that String has, but these seem to do for now.
 
     bool contains(UChar character) const { return m_string.contains(character); }
-    bool contains(const LChar* string) const { return m_string.contains(string); }
+    bool contains(ASCIILiteral literal) const { return m_string.contains(literal); }
     bool contains(StringView) const;
```

Comment:
Do we need to delete the overload that takes `const char*` to make sure no one uses that by accident when they intend to use them `ASCIILiteral` one? Isn’t that what was happening before?